src/Estimators/src/FloatingBaseEstimator.cpp

```diff
@@ -239,16 +240,20 @@ bool FloatingBaseEstimator::ModelComputations::getBaseStateFromIMUState(const iD
     A_H_B = A_H_IMU*(m_base_H_imu.inverse());
 
     // transform velocity (mixed-representation)
-    auto A_o_BIMU = A_H_B.getRotation().changeCoordFrameOf(m_base_H_imu.getPosition());
-    auto X = iDynTree::Transform(iDynTree::Rotation::Identity(), A_o_BIMU);
-    v_B = X*v_IMU;
+    auto A_o_BIMU = A_H_B.asSO3().act(m_base_H_imu.translation());
+    manif::SE3d X;
+    X.rotation().setIdentity();
+    X.translation(A_o_BIMU);
```

Comment:
I think you can directly call the constructor

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/215/commits/a7d54f75fdc83d383571d542bffd1481a3379771